.github/workflows/acceptance-tests-runner.yml

```diff
@@ -382,6 +382,38 @@ jobs:
             ./internal/service/advancedclustertpf
         run: make testacc
 
+  advanced_cluster_tpf_move:
+    needs: [ change-detection, get-provider-version ]
+    if: ${{ (needs.change-detection.outputs.advanced_cluster_tpf == 'true' || inputs.test_group == 'advanced_cluster_tpf') && inputs.terraform_version == '1.10.x' }}
+    env:
+      MONGODB_ATLAS_ADVANCED_CLUSTER_V2_SCHEMA: 'true'
+    strategy:
+      max-parallel: 2
+      fail-fast: false
+      matrix:
+        terraform_version: ['1.8.x', '1.9.x'] # TF supported versions for MoveState apart from latest one, 1.10.x
+    name: advanced_cluster_tpf_move-${{ matrix.terraform_version }}
+    runs-on: ubuntu-latest
+    permissions: {}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+        with:
+          ref: ${{ inputs.ref || github.ref }}
+      - uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed
+        with:
+          go-version-file: 'go.mod'
+      - uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd
+        with:
+          terraform_version: ${{ matrix.terraform_version }}
+          terraform_wrapper: false  
+      - name: Prepare new advanced_cluster
+        run: make tools enable-advancedclustertpf  
+      - name: Acceptance Tests
+        env:
+          ACCTEST_REGEX_RUN: '^TestAccAdvancedCluster_move'
```

Comment:
Are we missing a wildcard? Or is it only a single test? Or it works as a prefix?

Reply:
it's a prefix, for example the default value at the beginning of the file is:
```
ACCTEST_REGEX_RUN: ${{ inputs.test_name || inputs.provider_version == '' && '^Test(Acc|Mig)' || '^TestMig' }}
```